File: Polyglot/Vert.x/Hello/Hello.java
//
// A simple `hello-world` web server
//

import org.vertx.java.core.Handler;
import org.vertx.java.core.http.HttpServerRequest;
import org.vertx.java.platform.Verticle;

public class Hello extends Verticle {
    public void start() {
        vertx.createHttpServer().requestHandler(new Handler<HttpServerRequest>() {
            public void handle(HttpServerRequest req) {
                req.response().end("Hello World!");
            }
        }).listen(8080);
    }
}

// EOF

File: NodeJS/Learn/hello.js
#!/bin/sh
':' //; exec `command -v nodejs || command -v node || command -v js` "$0" "$@"

// hello.js - a simple `Hello, World!` web server from node's home page

var http = require('http');
http.createServer(function (req, res) {
    res.writeHead(200, {'Content-Type': 'text/plain'});
    res.end('Hello, World!\n');
}).listen(8081, 'localhost');
console.log('Server running at http://localhost:8081/');

// EOF
